webapp/src/assets/styles/theme.scss

```diff
@@ -11,21 +11,18 @@ $btn-border-radius-lg: .5rem;
 @import "~bootstrap/scss/_variables";
 @import "~bootstrap/scss/_mixins";
 
-// Base Styles
-@mixin mainContainer() {
+.main-container {
   max-width: 500px;
   margin: 0 auto;
 }
 
-.mainContainer {
-  @include mainContainer();
-}
-
-$custom-theme-colors: (
-  "suma": #c79866,
```

Comment:
Fewer variables are better. By adding 'suma' and 'darksuma', it meant we could not use 'primary' anywhere, which introduces significant problems. Additional colors are fine but we should be using BS's existing theming mechanism, which is replacing/setting values, not adding new ones to use everywhere. For example you can't use `variant=suma` for components.